flink-ai-flow/ai_flow/java/client/src/main/java/com/aiflow/client/MetadataClient.java

```diff
@@ -458,9 +459,10 @@ public ModelVersionMeta getModelVersionByVersion(String version, Long modelId) t
      * @param projectSnapshotId   Project snapshot id corresponded to model version.
      * @return Single ModelVersionRelationMeta object registered in Metadata Store.
      */
-    public ModelVersionMeta registerModelVersion(String modelPath, String modelType, String versionDesc, Long modelId, Long projectSnapshotId) throws Exception {
+    public ModelVersionMeta registerModelVersion(String modelPath, String modelType, String versionDesc, Long modelId, ModelVersionStage currentStage,
```

Comment:
IMO, the current stage is GENERATED when registering model version. Therefore this doesn't need to add `currentStage` parameter.

Reply:
This is deserved to be discussed, but we'd better keep it consistent with python client for now